README.md

```diff
@@ -1,26 +1,22 @@
-# 🔢 Week01 Alumni Project: Calculator
+# 🔢 Week01 Alumni Project: OOP Calculator
 
-### Goal: Build a Simple Calculator using JS OOP best practices
+This is a OOP concept calculator with a RESTFUL API to separate logic from front-end development.
```

Comment:
When writing an application, it is important to use the right tool for the job. In this case, a REST API is not the right tool. Calculations can be done right in the browser, no need to make a trip to the server to get the answer.